File: PRAKTIKUM MODUL 3/PRAK304_2410817310009_Muhamad_Ghazi_Rakhmadi.py
a = int(input("Input: "))

if a < 0 or a >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
elif a == 0:
    print("Nol")
elif 1 <= a <= 9:
    print("Satuan")
elif 11 <= a <= 19:
    print("Belasan")
elif a == 10 or a>=20 and a <= 99:
    print("Puluhan")
else:
    print("ERROR")

File: PRAKTIKUM MODUL 3/PRAK304_2410817310009_Muhamad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main () {
    int a;

    printf ("Input: ");
    scanf ("%d", &a);

    if (a == 0) {
        printf ("Output: Nol");
    } else if (a >= 1 && a <= 9) {
        printf ("Output: Satuan");
    } else if (a >= 11 && a <= 19) {
        printf ("Output: Belasan");
    } else if (a >= 10 || a >= 20 && a <= 99) {
        printf ("Output: Puluhan");
    } else {
        printf ("Output: ERORR");
    }

    return 0;
}